### PR TITLE
#252 backend update

### PR DESCRIPTION
**Pull request by kardan**
- [ ] Test plan | Unit test | Integration test
- [ ] Copyright header
- [ ] Code formatting
- [ ] Documentation

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@ pom.xml.asc
 *.class
 /.lein-*
 /.nrepl-port
+/.dir-locals.el
```
Comment (jonase) [inline backend/.gitignore]: there are some double entries in this `.gitignore` and the toplevel one. Perhaps we should have only one toplevel ignore file?

File: backend/project.clj [hunk 2]
```diff
@@ -55,16 +46,17 @@
    :uberjar       {:aot :all}
    :profiles/dev  {}
    :profiles/test {}
```
Comment (jonase) [inline backend/project.clj]: are these empty profiles needed?

File: backend/src/org/akvo/lumen/migrate.clj [hunk 3]
```diff
@@ -1,76 +1,71 @@
...
 
-           [([(_ :guard number?)] :seq)]
-           (rollback db [:tenants (first args)])
 
-           [([(_ :guard #(= % :tenants))
-              (_ :guard number?)] :seq)]
-           (doseq [tenant tenants]
-             (repl/rollback (tenant-spec tenant-spec-base tenant)
-                            (second args)))
+#_(defn do-rollback [datastore migrations]
```
Comment (jonase) [inline backend/src/org/akvo/lumen/migrate.clj]: are these rollback functions TODO or should they be removed?

File: backend/test/org/akvo/lumen/fixtures.clj [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
             [org.akvo.lumen.component.tenant-manager :as tm]
             [org.akvo.lumen.migrate :as migrate]
             [ragtime
              [jdbc :as jdbc]
              [repl :as repl]]
-            [reloaded.repl :refer [go stop]]
-            [user :refer [config]]))
+            [reloaded.repl :refer [go stop]]))
 
+#_(def config
```
Comment (jonase) [inline backend/test/org/akvo/lumen/fixtures.clj]: remove or keep?

File: backend/test/org/akvo/lumen/fixtures.clj [hunk 5]
```diff
@@ -51,7 +53,7 @@
 
 (hugsql/def-db-fns "org/akvo/lumen/fixtures.sql")
 
-(defn system-fixture
+#_(defn system-fixture
```
Comment (jonase) [inline backend/test/org/akvo/lumen/fixtures.clj]: remove or keep?

File: backend/src/org/akvo/lumen/migrate.clj (hunk 3 above)
Comment (kardan) [inline backend/src/org/akvo/lumen/migrate.clj]: I think we should have them. I'll add them

**Comment by jonase** [comment]
:+1: I've been testing this branch quite a lot and the update looks good to me.

**Pull request merged**